AutocompleteClientTests/FW/Logic/Worker/ConstructorIOQuizIntegrationTests.swift

```diff
@@ -117,7 +120,7 @@ class ConstructorIOQuizIntegrationTests: XCTestCase {
             let responseData = response.data!
 
             XCTAssertNil(cioError)
-            XCTAssertNotNil(responseData.versionId)
+            XCTAssertNotNil(responseData.quizVersionId)
```

Comment:
Should we add an assertion for `quizSessionId` as well for this test and the ones below?